src/app/containers/reference-form-dialog/reference-authors-list.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useCallback, SyntheticEvent } from 'react';
+import { TextField, IconButton, Menu, MenuItem } from '@material-ui/core';
+import { has, isEqual } from 'lodash';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import { SortableHandle, SortableContainer, SortableElement } from 'react-sortable-hoc';
+
+import { ReferencePerson } from 'app/models/reference';
+import { useReferenceAuthorStyles } from 'app/containers/reference-form-dialog/styles';
+import { ActionButton } from 'app/components/action-button';
+import DragIcon from 'app/assets/drag-indicator-grey.svg';
+
+interface ReferenceAuthorsListProps {
+  refAuthors: ReferencePerson[];
+  onChange(refAuthors: ReferencePerson[]): void;
+}
+
+interface RefAuthorInputProps {
+  refAuthor: ReferencePerson;
+  onDelete(): void;
+  onChange(refAuthors: ReferencePerson): void;
+}
+
+const labelProps = { shrink: true };
+
+const isGroupAuthor = (refAuthor: ReferencePerson) => {
+  return has(refAuthor, 'groupName');
+};
+
+const DragHandle = React.memo(
+  SortableHandle(() => <img src={DragIcon} alt="drag handle" aria-hidden={true} className="drag-handle" />),
+  isEqual
+);
+
+const SortableList = React.memo(
+  SortableContainer(({ children }) => <section> {children} </section>),
+  isEqual
+);
+
+const renderSingleAuthorForm = (classes, refAuthor, handleFormChange) => (
+  <>
+    <TextField
+      fullWidth
+      name="lastName"
+      label="Surname(s)"
+      classes={{ root: classes.lastName }}
+      InputLabelProps={labelProps}
+      variant="outlined"
+      value={refAuthor['lastName']}
+      onChange={handleFormChange}
+    />
+    <TextField
+      fullWidth
+      name="firstName"
+      label="Given name"
+      InputLabelProps={labelProps}
+      variant="outlined"
+      value={refAuthor['firstName']}
```

Comment:
This always is the first authors initial, but this doesn't look incorrect hence I guess the error is where you're extracting the info from the XML into the store when loading.